bench/wtperf/config.c

```diff
@@ -652,6 +678,72 @@ config_sanity(CONFIG *cfg)
 	return (0);
 }
 
+void
+config_queue_dedupe(CONFIG *cfg)
+{
+	CONFIG_QUEUE_ENTRY *conf_line, *test_line, *tmp;
+	char *string_key;
+
+	conf_line = TAILQ_FIRST(&cfg->config_head);
+	while (conf_line != NULL) {
+		string_key = strchr(conf_line->string, '=');
+		tmp = test_line = TAILQ_NEXT(conf_line, c);
+		while (test_line != NULL) {
+			if (strncmp(conf_line->string, test_line->string,
+			    string_key - conf_line->string + 1) == 0) {
```

Comment:
Can you verify that this works correctly when we have a config string that is a proper substring of another in all directions?  The only ones I know of at the moment is `max_latency` and `max_latency_fatal` and `min_throughput` and `min_throughput_fatal`.  That is, make sure having multiple ones of the shorter string and the longer string are consolidated properly.  I'm pretty sure the `+1` in the length forces it to match the `=` too.  That is a good comment to add.